addons/nvidia-bootstrap/image/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# Simple early detection of nvidia card
+grep 10de102d /proc/bus/pci/devices || exit 0
```

Comment:
If you want to allow use of GPUs on G2 instances, this should work:
`egrep '10de118a|10de102d' /proc/bus/pci/devices || exit 0`